keyboards/3.0/fr-t-k0-optimise.xml

```diff
@@ -0,0 +1,230 @@
+		<key id="ellipsis" output="…" />
+		<key id="open-apos" output="‘" />
+		<key id="close-apos" output="’" />
+		<key id="open-g" output="«" />
+		<key id="close-g" output="»" />
+
+		<key id="theta" output="θ" /> <!-- "for Romani" -->
+		<key id="Theta" output="ϴ" /> <!-- "for Romani" -->
+		<key id="tm" output="™" />
+		<key id="open-double" output="“" />
+		<key id="close-double" output="”" />
+		<key id="oe" output="œ" />
+		<key id="Oe" output="Œ" />
+		<key id="registered" output="®" />
+		<key id="minus-sign" output="−" />
+		<key id="ss" output="ß" />
+		<key id="ininity" output="∞" />
+		<key id="division" output="÷" />
+		<key id="less-equal" output="≤" />
+		<key id="greater-equal" output="≥" />
+		<key id="copy" output="©" />
+		<key id="inverse-question" output="¿" />
+		<key id="inverse-bang" output="¡" />
+		<key id="middle-dot" output="•" />
+		<key id="A-grave" output="À" />
+		<key id="E-acute" output="É" />
+		<key id="E-caret" output="Ê" />
+		<key id="em-dash" output="—" />
+		<key id="open-single" output="’" />
+		<key id="close-single" output="’" />
+		<key id="u-grave" output="Ù" />
+		<key id="radix" output="√" />
+		<key id="C-cedilla" output="Ç" />
+		<key id="not-equal" output="≠" />
+        <key id="open-angle-quote" output="‹" />
+        <key id="close-angle-quote" output="›" />
+
+		<key id="infinity" output="∞" />
+		<key id="E-grave" output="È" />
+		<key id="U-grave" output="Ù" />
+		<key id="nb-hyphen" output="‑" />
+		<key id="multiplication" output="×" />
+		<key id="ezh" output="ʒ" />
+		<key id="Ezh" output="Ʒ" />
+		<key id="ae" output="æ" />
+		<key id="Ae" output="Æ" />
+		<key id="permille" output="‰" />
+		<key id="dbl-dagger" output="‡" />
+		<key id="dagger" output="†" />
+		<key id="asym-equal" output="≃" />
+		<key id="Ss" output="ẞ" />
+        <!-- interestingly, this key has no output by itself, but is U+00A4 when preceded by the currency mark. -->
+        <key id="altshifte" output="\m{altshifte}"/>
+	</keys>
+
+	<layers formId="iso">
+		<layer modifiers="none">
+			<row
+				keys="at a-grave e-acute e-grave e-caret open-paren close-paren open-apos close-apos open-g close-g apos mark-caret" />
+			<row keys="a z e r t y u i o p hyphen plus" />
+			<row keys="q s d f g h j k l m slash asterisk" />
+			<row keys="open-angle w x c v b n period comma colon semi-colon" />
+			<row keys="space" />
+		</layer>
+
+		<layer modifiers="shift">
+			<row keys="hash 1 2 3 4 5 6 7 8 9 0 double-quote mark-umlaut" />
+			<row keys="A Z E R T Y U I O P en-dash plus-minus" />
+			<row keys="Q S D F G H J K L M backslash vulgar-half" />
+			<row keys="close-angle W X C V B N question bang ellipsis equal" />
+			<row keys="space" />
+		</layer>
+        <!-- layer is for AltGr - for now we use ctrl alt -->
+		<layer modifiers="ctrl alt">
+			<row
+				keys="mark-breve section mark-acute mark-grave amp open-square close-square mark-macron underscore open-double close-double degree mark-caron" />
```

Comment:
The caron dead key (`mark-caron`) must also grant access to subscript numerals (0-9).